{{cookiecutter.project_slug}}/requirements.txt

```
@@ -1,5 +1,5 @@
 {%- if 'mssql' == cookiecutter.db_driver_name %}
-git+https://github.com/jssuzanne/sqlalchemy-utils.git@mssql_and_pymssql#egg=sqlalchemy_utils
+SQLAlchemy-Util
```

I don't belive this library exits on pypi: https://pypi.org/project/SQLAlchemy-Util/ do you means [SQLAlchemy-Utils](https://pypi.org/project/SQLAlchemy-Utils/)
```suggestion
SQLAlchemy-Utils
```